Hanan-Qays/style.css

```diff
@@ -0,0 +1,95 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+  body {
+      display:flex;
+      height:100%;
+      font-family:Arial, Helvetica, sans-serif;
+    }
+    aside {
+      background-color:#3d4f5d;
+      flex: 1;
+      width: 15%;
+      height: 100vw;
```

Comment:
we use **vh** for height and **vw** for width
```
width: 100vw;
height: 100vh;
```

Reply:
thanks ,but when i write this code i don't know we have something called vh and vw